client/app/scripts/superdesk-desks/desks.js

```diff
@@ -888,17 +888,22 @@
                     };
 
                     function errorMessage(response) {
-                        if (response.data && response.data._issues && response.data._issues.name && response.data._issues.name.unique) {
-                            scope._errorUniqueness = true;
-                        } else {
-                            scope._error = true;
+                        scope._error = true;
+                        scope._errorMessage = gettext('There was a problem, desk not created/updated.');
+
+                        if (response.data && response.data._issues) {
+                            if (response.data._issues.name && response.data._issues.name.unique) {
+                                scope._errorMessage = gettext('Desk with name ' + scope.desk.edit.name + ' already exists.');
```

Comment:
this would require translation string for every desk name, but you can use `$interpolate` service:
```
msg = $interpolate(gettext('Desk with name {{ name }} already exists.'))({name: scope.desk.edit.name});
```